src/test/mon/test-mon-msg.cc

```diff
@@ -224,8 +224,8 @@ class MonMsgTest : public MonClientHelper,
   Cond cond;
 
   MonMsgTest() :
-    reply_msg(NULL),
     MonClientHelper(g_ceph_context),
+    reply_msg(NULL),
     lock("lock") { }
 
```

Comment:
I think it's easier in the long run if, for things like initializing pointers to null and the like, we just use a default value for the class member.

(With that convention things that SHOULD be initialized obviously are or aren't, and reorder takes care of itself.)